### PR TITLE
add medium to user tag and profile other-sites block

### PR DESCRIPTION
**Pull request by srukle**
This addresses issue #1997.

Sorry, the past PR was documented incorrectly. Thank you for pointing that out @afuna.

### CODE REVIEW

File: cgi-bin/DW/Logic/ProfilePage.pm [hunk 1]
```diff
@@ -838,6 +838,17 @@ sub external_services {
         };
     }
 
+    if ( my $emedium = $u->prop( 'medium' ) ) {
```
Comment (zorkian) [inline cgi-bin/DW/Logic/ProfilePage.pm]: This should be `my $medium = ...` reason being you need to have the raw contents so you can escape it for HTML later and for URL both.

File: cgi-bin/DW/Logic/ProfilePage.pm [hunk 2]
```diff
@@ -838,6 +838,17 @@ sub external_services {
...
+	my $emedium = LJ::eurl( $emedium );
+	push @ret, {
+	    type => 'medium',
+	    text => LJ::ehtml( $emedium ),
```
Comment (zorkian) [inline cgi-bin/DW/Logic/ProfilePage.pm]: You don't want to double-escape something -- in this case you eurl'd the contents and then you're ehtml'ing the eurl'd comments which won't work out well.

**Comment by kareila** [comment]
I just noticed that this version of the pull request is missing the necessary changes to htdocs/manage/profile/index.bml.

**Comment by srukle** [comment]
Does @kaberett's fork resolve these problems?
https://github.com/kareila/dw-free/commit/3aa01daad73520f755073e8e80d901c3725718b3

**Comment by zorkian** [comment]
Pinging @kareila on the question!

**Comment by kareila** [comment]
Sorry - I'm not sure which fork you mean. The link you pasted was to my work for the user tag.

If you look at #2008 you'll see there are changes there to htdocs/manage/profile/index.bml that were not included here.

Also, this PR now has a merge conflict due to overlapping changes in htdocs/profile.bml.text.

Please address these issues and also @zorkian's requested changes.

**Comment by zorkian** [comment]
This is old -- closing for now! If you ever have time/want to pick it up again, would love to have you!

**Pull request closed without merging**